3p/3p.js

```diff
@@ -82,9 +79,7 @@ export function run(id, win, data) {
  * @param {function()=} opt_cb
  */
 export function writeScript(win, url, opt_cb) {
-  /*eslint no-useless-concat: 0*/
-  win.document
-      .write('<' + 'script src="' + encodeURI(url) + '"><' + '/script>');
+  win.document.write('<script src="' + encodeURI(url) + '"></script>');
```

Comment:
This would be problematic if we inlined the script, because it would terminate the outer `<script>` element. But it is probably OK.

Reply:
Good catch. String concatenations aren't automatically done by Prettier; this was the result of a find-replace done by me.

I've reverted the two string concatenations in this PR that contain a `<script>` tag, and exempted them using `eslint-disable-next-line`.